docker-jans-casa/Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN mkdir -p ${JETTY_BASE}/jans-casa/webapps \
 
 # casa plugins
 RUN mkdir -p ${JETTY_BASE}/jans-casa/plugins \
-    && for casa_plugin in acct-linking authorized-clients bioid-plugin custom-branding email_2fa_core strong-authn-settings; \
+    && for casa_plugin in acct-linking-agama acct-linking authorized-clients bioid-plugin custom-branding strong-authn-settings; \
```

Comment:
@moabu acct-linking-agama plugin is not following the same naming suffix (`-jar-with-dependencies.jar`) so it will not be downloaded. Also, the docs at https://docs.jans.io/v1.1.5/casa/plugins/accts-linking/account-linking-index/#components-deployment mentions about deploying acct-linking-agama component using TUI.

Reply:
From CN context, we4 need to allow user to change `-Dacr` in entrypoint.